packages/cli/templates/default-backend-module/src/index.ts.hbs

```diff
@@ -0,0 +1,8 @@
+/***/
```

Comment:
remove

Reply:
It needs to be there to make sure the linter doesn't remove the package comment 😅